ckanext/canada/helpers.py

```diff
@@ -773,6 +775,40 @@ def date_field(field, pkg):
         return pkg.get(field).split(' ')[0]
     return pkg.get(field, None)
 
+def registry_network_access(remote_addr):
+    """
+    Only allow requests from GOC network to access
+    user account registration view
+   """
+    try:
```

Comment:
@RabiaSajjad just above the try, you may need to add:
```
remote_addr = text_type(remote_addr)
```
to convert bytes and strings to correct types that the ipaddress module expects